MediaManager.Forms/Xaml/StopExtension.cs

```diff
@@ -0,0 +1,19 @@
+﻿using MediaManager.Playback;
+
+namespace MediaManager.Forms.Xaml
+{
+    public class StopExtension : MediaExtensionBase
+    {
+        protected override bool CanExecute() => 
+            MediaManager.State == MediaPlayerState.Playing;
+
+        protected override void Execute()
+        {
+            MediaManager.Stop();
+            MediaManager.PlayPreviousOrSeekToStart();
```

Comment:
Should this not only contain Stop?

Reply:
yes looks like I missed that in the cleanup... thanks for the catch.